packages/story-editor/src/components/library/panes/media/common/mediaElement.js

```diff
@@ -271,9 +271,14 @@ Element.propTypes = {
  * @param {string?} props.margin The margin in around the element
  * @param {Function} props.onInsert Insertion callback.
  * @param {string} props.providerType Which provider the element is from.
+ * @param props.canEditMedia
```

Comment:
Needs type + description.

Reply:
Fixed in https://github.com/GoogleForCreators/web-stories-wp/pull/12949/commits/617a13c5c62304d6f4fd3454569fac9fb4e2df3d